src/index.html

```diff
@@ -11,12 +11,98 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
+    <header class="header-container">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
```

Comment:
The `<img>` tag for the logo lacks a descriptive `alt` attribute. Consider providing a more descriptive text for better accessibility, such as 'Company Logo'.